docker-for-mac/osxfs.md

```diff
@@ -1,369 +0,0 @@
-object is told that its `uid` and `gid` own the object. When any containerized
-process changes the ownership of a shared file system object, such as by using
-the `chown` command, the new ownership information is persisted in the
-`com.docker.owner` extended attribute of the object. Subsequent requests for
-ownership metadata return the previously set values. Ownership-based permissions
-are only enforced at the macOS file system level with all accessing processes
-behaving as the user running Docker. If the user does not have permission to
-read extended attributes on an object (such as when that object's permissions
-are `0000`), `osxfs` attempts to add an access control list (ACL) entry that
-allows the user to read and write extended attributes. If this attempt fails,
-the object appears to be owned by the process accessing it until the extended
-attribute is readable again.
-
-### File system events
-
-Most `inotify` events are supported in bind mounts, and likely `dnotify` and
-`fanotify` (though they have not been tested) are also supported. This means
-that file system events from macOS are sent into containers and trigger any
-listening processes there.
-
-The following are **supported file system events**:
-
-* Creation
-* Modification
-* Attribute changes
-* Deletion
-* Directory changes
-
-The following are **partially supported file system events**:
-
-* Move events trigger `IN_DELETE` on the source of the rename and
-  `IN_MODIFY` on the destination of the rename
-
-The following are **unsupported file system events**:
-
-* Open
-* Access
-* Close events
-* Unmount events (see [Mounts](#mounts))
-
-Some events may be delivered multiple times. These limitations do not apply to
-events between containers, only to those events originating in macOS.
-
-### Mounts
-
-The macOS mount structure is not visible in the shared volume, but volume
-contents are visible. Volume contents appear in the same file system as the rest
-of the shared file system. Mounting/unmounting macOS volumes that are also bind
-mounted into containers may result in unexpected behavior in those containers.
-Unmount events are not supported. Mount export support is planned but is still
-under development.
-
-### Symlinks
-
-Symlinks are shared unmodified. This may cause issues when symlinks contain
-paths that rely on the default case-insensitivity of the default macOS file
-system.
-
-### File types
-
-Symlinks, hardlinks, socket files, named pipes, regular files, and directories
-are supported. Socket files and named pipes only transmit between containers and
-between macOS processes -- no transmission across the hypervisor is supported,
-yet. Character and block device files are not supported.
-
-### Extended attributes
-
-Extended attributes are not yet supported.
-
-### Technology
-
-`osxfs` does not use OSXFUSE. `osxfs` does not run under, inside, or
-between macOS userspace processes and the macOS kernel.
-
-### SSH agent forwarding
```

Comment:
Same for this (which I don't think is specific to osxfs being the implementation)